src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
```

Comment:
The title should be independent of the breadcrumbs :).

Reply:
OK fixed 6b53cce 2d72a91